macros.in

```diff
@@ -745,12 +745,14 @@ package or when debugging this package.\
 %_pkgverify_flags 0x0
 
 # Set to 1 to minimize writing (at the cost of more reads) to
-# conserve eg SSD disks.
-%_minimize_writes      0
+# conserve eg SSD disks. Set to -1 for enabling when runnning on SSDs.
+# Set to 0 for always off.
+%_minimize_writes      -1
```

Comment:
This will cause %_minimize_writes to be enabled by default because of how rpmtsCreate() initializes - any non-zero value will enable it. Which is not right, -1 should probably mean "autodetect", and 0 is explicit disable.